src/shared/src/types.rs

```diff
@@ -28,8 +28,18 @@ pub mod transaction {
     }
 }
 
+pub type Timestamp = u64;
+
```

Comment:
Is the idea that when a client requests a change, you wish to make sure that the client has an up-to-date version of the account?  Essentially the timestamp is being used as a mutation counter.  Is that right?  If so, I think there might be  a problem in that `time()` returns the same value within a block, so multiple changes within a single block may all store the same time.  At least theoretically timestamps are not adequate, although they may be fine in practice.  It would be sounder to use a change counter.  But let's make sure I understand the goal first. :-D

Reply:
I would write `changecount = changecount.unwrap_or_default() + 1;`.  That would be enough.  But _if_ you would find the timestamp useful in other ways we can still look at whether block timestamp is enough.

Reply:
Ah gotcha! No sure let's do that, when you wrote "change counter" I assumed you were expected something more complicated. I'll implement a `changecount`. I might name the variable `version`, if I reckon correctly in few SQL projects that was the name of the field used for such purpose. Will do and ping you when done.

Reply:
i.e. I'll modify this PR.